charts/pravega/templates/clusterrole.yaml

```diff
@@ -2,7 +2,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ .Values.serviceAccount.name }}
+  name: {{ template "pravega.fullname" . }}-{{ .Release.Namespace }}
   namespace: {{ .Release.Namespace }}
```

Comment:
ClusterRole is not a namespaced resource. Any idea how the `namespace` metadata got here?

Reply:
Thanks for pointing it, I can remove it

Reply:
Again, if this role is needed to deploy pravega cluster in one specific namespace, it should not be a ClusterRole at all. 
A namespaced `Role` resource should be enough here.

Reply:
Roles are there in addition to clusterrole

Reply:
Sorry, did not notice it needed access to `nodes`.